src/content/en/updates/2018/07/css-scroll-snap.md

```diff
@@ -0,0 +1,365 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Introduction to CSS Scroll Snap.
+
+
+{# wf_updated_on: 2018-07-25 #}
+{# wf_published_on: 2018-07-24 #}
+{# wf_tags: chrome69,css,snap,scroll #}
+{# wf_blink_components: Blink>CSS #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_featured_snippet: CSS Scroll Snap allows web developers to create well-controlled scroll experiences by declaring scroll snapping positions. This enables common UX scroll patterns without the need for JavaScript. #}
+
+# Well-Controlled Scrolling with CSS Scroll Snap {: .page-title }
+
+{% include "web/_shared/contributors/flackr.html" %}
+{% include "web/_shared/contributors/majidvp.html" %}
+
+
+<style>
+figure {
+  text-align: center;
+}
+figcaption {
+  font-size: 14px;
+  font-style: italic;
+}
+</style>
+
+
+## TL;DR {: #tldr .hide-from-toc}
+
+[CSS Scroll Snap](https://drafts.csswg.org/css-scroll-snap/) feature allows web
+developers to create well-controlled scroll experiences by declaring scroll
+snapping positions. Paginated articles and image carousels are two commonly used
+examples of this. CSS Scroll Snap provides an easy to use and consistent API for
+building these popular UX patterns and Chrome is shipping a high fidelity and
+fast implementation of it in version 69.
+
+
+## Background {: #background}
+
+### The case for scroll snapping
+
+Scrolling is a popular and natural way to interact with content on the web. It
+is the platform's native means of providing access to more information than is
+visible on the screen at once, becoming especially vital on mobile platforms
+with limited screen real estate. So it is no surprise that web authors
+increasingly prefer to organize content into scrollable flat lists as opposed to
+deep hierarchies.
+
+Scrolling's main drawback is its lack of precision. Rarely does a scroll end up
+aligned to a paragraph or sentence. This is even more pronounced for paginated
+or itemized content with meaningful boundaries when the scroll finishes at the
+middle of the page or image leaving it partially visible. These use cases
+benefit from a well-controlled scrolling experience.
+
+Web developers have long relied on Javascript based solutions for controlling
```

Comment:
JavaScript should be capitalized as such, same on line 58

Reply:
Done